test/verify/check-machines-virsh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/python2
+# -*- coding: utf-8 -*-
+
+# This file is part of Cockpit.
+#
+# Copyright (C) 2018 Red Hat, Inc.
+#
+# Cockpit is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation; either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# Cockpit is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+
+import parent
+import unittest
+
+from testlib import *
+import machineslib
+
+
+class TestMachinesVirsh(machineslib.TestMachines):
+    def removeLibvirtDBus(self):
+        m = self.machine
+        if ("debian" in self.machine.image) or ("ubuntu" in self.machine.image):
+            m.execute("dpkg --remove libvirt-dbus || true")
+        else:
+            m.execute("rpm -e libvirt-dbus || true")
```

Comment:
Do these commands regularly fail? If not, I suggest to remove the alternative truth, to fail more obviously if these ever get renamed, or our images get broken somehow. Otherwise this test case could silently test the dbus case again.

Reply:
Thanks, makes sense.